ddl/reorg.go

```diff
@@ -273,21 +262,21 @@ func (w *worker) runReorgJob(t *meta.Meta, reorgInfo *reorgInfo, tblInfo *model.
 		}
 	case <-w.ctx.Done():
 		logutil.BgLogger().Info("[ddl] run reorg job quit")
-		w.reorgCtx.setNextKey(nil)
-		w.reorgCtx.setRowCount(0)
-		w.reorgCtx.resetWarnings()
+		rc.setNextKey(nil)
+		rc.setRowCount(0)
+		rc.resetWarnings()
```

Comment:
We remove `reorgCtx.clean()`, so cloud we can using `removeReorgCtx` instead of these lines?

Reply:
good point!